src/fidesctl/cli/commands/generate.py

```diff
@@ -74,6 +75,47 @@ def generate_dataset_db(
     )
 
 
+@generate_dataset.command(name="bigquery")
+@click.pass_context
+@click.argument("dataset_name", type=str)
+@click.argument("output_filename", type=str)
+@credentials_id_option
+@click.option("--keyfile-path", type=str, default=".fides/bigquery.json")
```

Comment:
This has a default value yet the `handle_bigquery_config_options` function has logic to account for an empty keyfile path when using `credentials_id`

Reply:
great catch, thank you 🙌🏽